compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/SymbolEnter.java

```diff
@@ -481,6 +530,18 @@ private void checkErrors(BLangTypeDefinition unresolvedType, BLangType currentTy
                     }
                 }
             }
+        } else if (currentTypeNode.getKind() == NodeKind.ENDPOINT_TYPE) {
```

Comment:
Since we have many else if blocks where we don't do anything, shall we merge them to one else if block?